awx/playbooks/project_update.yml

```diff
@@ -159,23 +159,29 @@
   gather_facts: false
   connection: local
   name: Install content with ansible-galaxy command if necessary
+  vars:
+    yaml_exts:
+      - {ext: .yml}
+      - {ext: .yaml}
   tasks:
 
     - block:
-        - name: detect requirements.yml
+        - name: detect requirements.(yml/yaml)
```

Comment:
minor naming comment - this should say "detect _roles/_ requirements.(yml/yaml)", like the other one for collections